### DISCUSSION TITLE
Add a lightbox effect to images in the document

### DISCUSSION THREAD

**Discussion by Muzych** [withastro/starlight#1554] [Feature Requests]
### What version of `starlight` are you using?

0.18.0

### What is your idea?

I think it is necessary to add a lightbox effect to the pictures in the document. When users upload images of inappropriate sizes, the results on the web page will be poor. Adding lightbox effects to images can  definitely increase user experience.

### Why is this feature necessary?

Same as above.

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by HiDeoo** [comment]
Thanks for your feedback.

Such feature is currently [available as Starlight plugin](https://github.com/HiDeoo/starlight-image-zoom) which adds zoom capabilities to documentation images.
